client/src/components/utils/PopUp/style.css

```diff
@@ -0,0 +1,62 @@
+.popup{
+  position: absolute;
+  text-align: center;
+  width: 80%;
```

Comment:
`Suggestion` you can use vw vh 
```suggestion
  width: 80vw;
```

Reply:
-_-